src/start.ts

```diff
@@ -154,13 +154,14 @@ export const handler = async (_event: any = {}): Promise<any> => {
 
             for (const {
               templateId,
-              parameters,
+              overrideParameters,
+              templateParameters,
               oisTitle,
               endpointName,
               deviationPercentage,
               requestSponsor,
             } of rrpBeaconServerKeeperJobs) {
-              const encodedParameters = abi.encode(parameters);
+              const encodedParameters = abi.encode(overrideParameters);
```

Comment:
This may be incorrect: this output is used in the generation of the `beaconId`, so it should be the sum of all parameters (template and on-chain)?
In most scenarios `overrideParameters` will be an empty array.

Reply:
You are correct Aaron. Previously I was getting the whole list of parameters from a single object called parameters and when I refactored into 2 separate arrays is when I made this mistake. Thanks!